src/coreclr/src/System.Private.CoreLib/src/System/Reflection/RuntimeModule.cs

```diff
@@ -91,17 +91,9 @@ public override byte[] ResolveSignature(int metadataToken)
         [RequiresUnreferencedCode("Trimming changes metadata tokens")]
         public override MethodBase? ResolveMethod(int metadataToken, Type[]? genericTypeArguments, Type[]? genericMethodArguments)
         {
-            MetadataToken tk = new MetadataToken(metadataToken);
-
-            if (!MetadataImport.IsValidToken(tk))
-                throw new ArgumentOutOfRangeException(nameof(metadataToken),
-                    SR.Format(SR.Argument_InvalidToken, tk, this));
-
-            RuntimeTypeHandle[]? typeArgs = ConvertToTypeHandleArray(genericTypeArguments);
-            RuntimeTypeHandle[]? methodArgs = ConvertToTypeHandleArray(genericMethodArguments);
-
             try
             {
+                MetadataToken tk = new (metadataToken);
```

Comment:
Nit: I don't think we use implicitly typed _new_ throughout the codebase. (Same comment elsewhere throughout this PR.)

Reply:
Changed